.gitignore

```diff
@@ -35,3 +38,8 @@ _testmain.go
 
 vendor/
 .bin/
+
+# Assorted tool configuration
+.envrc
```

Comment:
These are direnv file right, you could always add it to the global github ignore on your machine, then you wouldn't need to add them to the repo specific ones.

Reply:
Yeah - I think I already do have that. I added these because goreleaser refuses to run if the local checkout is dirty in any way, and it doesn't see my global config inside of act.
There's an option to disable it, but it seems to turn off sanity checks on the git history for the build that seemed useful.
I'll remove these though, they were really only needed for local testing of the workflow.